packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -223,3 +223,19 @@ ButtonCTAVariants = () => (
   </React.Fragment>
 );
 ```
+
+### Progress
+```js
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+
+ButtonProgressVariants = () => (
+  <React.Fragment>
+    <Button spinnerAriaValueText="Not loading" isLoading={false} variant="primary">Primary loader</Button>{' '}
```

Comment:
can remove the `spinnerAriaValueText` props when isLoading is false since it won't get rendered anyways.